src/katgpucbf/fgpu/send.py

```diff
@@ -61,7 +63,7 @@ class Frame:
     timestamp
         Zero-dimensional array of dtype ``>u8`` holding the timestamp.
     data
-        Payload data for the frame, of shape (channels, spectra_per_heap, N_POLS, COMPLEX).
```

Comment:
Does the `heap_data_shape` in `make_descriptor_heap` also need to be updated? Line 299.

Reply:
No, because that's specified by the ICD, and it uses int8 as the dtype rather than the (real, imag) struct. SPEAD supports sub-byte types (like int4) so it's not necessary to change anything there.

Reply:
Too right, too right.